### DISCUSSION TITLE
Advise related to fryefryefrye/Open-Source-RKS repo

### DISCUSSION THREAD

**Discussion by Axxo250** [community/community#126008] [Programming Help]
Hi All,
So im fairly new to Git Hub and  microcontrollers. I hope I have posted my question in the right section on the forum. If I haven't please let me know. I have been playing around with verious components and software lately to try and get a better understanding of programming and electronics as a whole.
 learning a language seems to be quite tricky for me.

Anyway I would be really greatfull if someone could help point me in the right direction with understanding how to implicate this repo fryefryefrye/Open-Source-RKS .
Im trying to use my android phone to trigger the nano at different RF levels using a nrf24 module and nano.

 I've  managed to get my IRK number using a esp32 from one of the included examples however I can't seem to work out how to get the phone to trigger the nano using the RX example included.  there seems to be so many veriables I don't understand when reading data signals using the nordic nfr app and want to make sure I'm not compremising my WiFi security etc by playing around with the app. 
And help would be great and let me know if you need more info?

Cheers 
Chris

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Axxo250 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by Axxo250** [reply]
UPDATE:
I've now managed to get my MAC address advertising randomly but im unsure how to implicate this into usable code to pull the relay HIGH or LOW when I walk close to car and vice versa. please help??

**Comment by fryefryefrye** [comment]
Please check https://github.com/fryefryefrye/Decoding-Random-Bluetooth-Address
And use the examples "nrf_ble_add" get the phone to trigger the nano.
